src/Send/ConfirmScreen/ConfirmScreen.tsx

```diff
@@ -24,7 +23,7 @@ export type Params = {
   balanceAfterTx: BigNumber
```

Comment:
can we convert it to string? params should have only serializable data.

Reply:
Done. Thanks.